src/components/MdText.vue

```diff
@@ -0,0 +1,43 @@
+<script>
+import MarkdownIt from 'markdown-it'
+
+export default {
+    name: 'MdText',
+    props: {
+        source: {
+            type: String,
+            required: true
+        },
+        tag: {
+            type: String,
+            default: 'div'
+        }
+    },
+    data() {
+        return {
+            md: null
+        }
+    },
+    computed: {
+        content() {
+            const src = this.source
+            return this.md?.render(src)
+        }
+    },
+    created() {
+        this.md = new MarkdownIt()
+        this.md.options.breaks = true
+    },
+    destroyed() {
+        this.md = null
+    },
+    render(h) {
+        const paragraphs = this.content.replaceAll('<br>', '</p><p>')
+        return h(this.tag, { domProps: { innerHTML: paragraphs } })
+    }
```

Comment:
With this, I'm afraid that if we have the automatic merging of the translations, we introduce a security problem with `v-html`: the paragraphs will be provided by translators in Transifex. So, I think it should stay as it is. 

I didn't know about the render effect on the DOM updates, thank you!

Reply:
`v-html` does the same thing as setting `innerHTML` (source: [StackOverflow](https://stackoverflow.com/a/53320455/2601645)). If it's just text, you can also use `v-text`.

Reply:
This is something I've read in the [Vue official docs](https://vuejs.org/v2/guide/syntax.html#Raw-HTML)

> Dynamically rendering arbitrary HTML on your website can be very dangerous because it can easily lead to XSS vulnerabilities. Only use HTML interpolation on trusted content and never on user-provided content.

Then I went down the rabbit hole and found a way to display an alert in the Chooser using `v-html`:

```vue
<div class="help-text" v-html="<img src='x' onerror='alert(1)'>" />
```
The example is actually from [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML).
So, I guess we should leave it as is, and not use v-html, if even I could do this :)